src/modules/project/pages/projectView/views/projectMainBody/components/MobileMilestoneRecap.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, HStack, Text, VStack } from '@chakra-ui/react'
+import { useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { Body1 } from '../../../../../../../components/typography'
+import { useProjectMilestones } from '../../../../../pages/projectView/hooks/useProjectMilestones'
+
+export const MobileMilestoneRecap = () => {
+  const { t } = useTranslation()
+
+  const { currentMilestone, milestoneIndex, prevMilestone, balance } = useProjectMilestones()
+
+  const getMilestoneValue = useCallback(() => {
+    if (!currentMilestone) return null
+
+    return Math.ceil(((balance - prevMilestone) / (currentMilestone.amount - prevMilestone)) * 100)
+  }, [balance, currentMilestone, prevMilestone])
+
+  function formatAmount(amount: number) {
+    if (amount < 10000) {
+      return `${amount} sats`
+    }
+
+    if (amount < 1000000) {
+      return `${amount / 1000}K sats`
+    }
+
+    const millions = amount / 1000000
+
+    if (millions < 10) {
+      return `${millions.toFixed(2)}M sats`
+    }
+
+    if (millions < 100) {
+      return `${millions.toFixed(1)}M sats`
+    }
+
+    return `${Math.round(millions)}M sats`
+  }
```

Comment:
I think we already have a function for this, getShortAmountLabel in utils , you can have it with out without a digit after decimal.

Reply:
Thanks!